js/models/conversations.js

```diff
@@ -1109,37 +1101,22 @@
                 expireTimer,
                 profileKey,
                 options
-              )
+              ),
+              message.isFriendRequest()
             )
           )
         );
 
         return true;
       });
     },
-    async updateTextInputState() {
-      // Check if we need to disable the text field
-      if (!this.isFriend()) {
-        // Disable the input if we're waiting for friend request approval
-        const waiting = await this.waitingForFriendRequestApproval();
-        if (waiting) {
-          this.trigger('disable:input', true);
-          this.trigger('change:placeholder', 'disabled');
-          return;
-        }
-        // Tell the user to introduce themselves
-        this.trigger('disable:input', false);
-        this.trigger('change:placeholder', 'friend-request');
-        return;
-      }
-      this.trigger('disable:input', false);
-      this.trigger('change:placeholder', 'chat');
-    },
-    wrapSend(promise) {
+    wrapSend(promise, isFriendRequest = false) {
```

Comment:
where is `wrapSend` used with `isFriendRequest` set ?
Is there a way to get that information via `this` instead?